ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/transactions/PendingTransactions.java

```diff
@@ -92,6 +99,20 @@ public PendingTransactions(
             "operation");
   }
 
+  public void evictOldTransactions() {
+    synchronized (pendingTransactions) {
+      long now = clock.millis();
```

Comment:
nit: final
Even more nitty: :) Personally I would have made this `final Instant removeTransactionsBefore = clock.instant().minusMillis(transactionEvictionIntervalMs)` and then the filter would be `transaction -> transaction.getAddedToPoolAt().isBefore(removeTransactionsBefore)`. Both ways work fine and it's not a big deal but I find the clarity of calculating the cut-off timepoint helpful.